app/domain/authentication/authn_jwt/configuration_jwt_generic_vendor.rb

```diff
@@ -2,6 +2,13 @@ module Authentication
   module AuthnJwt
     # Mock JWTConfiguration class to use it to develop other part in the jwt authenticator
     class ConfigurationJWTGenericVendor < ConfigurationInterface
+
+      def self.validate_input(authentication_parameters)
+        Authentication::AuthnJwt::ValidateInput.new.call(
```

Comment:
the `Authentication::AuthnJwt::ValidateInput.new` should be in initialize function and in the interface function you only execute the call function